src/components/page-seo/page-seo.tsx

```diff
@@ -0,0 +1,19 @@
+import Head from "next/head";
```

Comment:
I think that the `Head` part should be implemented only when a relevant use case requires it. I understand that React merges all Head content into a single `head` element, but there's no reason to rely on this feature in contexts that are already scoped to a `Head` component.

Reply:
So each component would declare its own `Head`, or rather use a prop to wrap everything in `Head`?

On second thought - why? It's an SEO enhancement that uses `Head` as its env; so why separate?

Reply:
I wrote that in the current implementation, as far as I can see, the page-seo is used in a contexts that already has a head.  If you see a different real-world case on the horizon, then you should leave the current impl.